packages/react/src/input/Input.css.ts

```diff
@@ -6,13 +6,14 @@ export const wrapper = recipe({
     {
       alignItems: "center",
       border: "1",
-      borderColor: "border.default",
       color: "fg.default",
       display: "flex",
       flexDirection: "row",
       rounded: "sm",
     },
     style({
+      borderColor: "border.default",
```

Comment:
Needs to be fetched from `theme`
```suggestion
      borderColor: theme.colors["border.default"],
```

Reply:
todo: write a lint rule to catch these